packages/terra-date-time-picker/src/DateTimePicker.jsx

```diff
@@ -262,7 +263,8 @@ class DateTimePicker extends React.Component {
       dateTime: newDateTime,
     });
 
-    if (this.props.onChange) {
+    // We want to not fire onChange in the event that the new time is ambiguous and the old time was not. This way the user can use TimeClarification before we fire the onChange function.
```

Comment:
nitpick - want to make this more concise?
```suggestion
    // If the new time is ambiguous and the old time is not, do not fire onChange. This allows a user to use TimeClarification before onChange is fired.
```

Reply:
Fixed https://github.com/cerner/terra-framework/pull/555/commits/32443749bb26dca02597dd2efc4dc95cea1efcba